assets/src/styles/components/_network.scss

```diff
@@ -0,0 +1,188 @@
+#network-map {
+  height: 350px;
+  margin: 1em 0;
+
+  .legend {
+    padding: 2px;
+    background-color: white;
+
+    .legend-expand-container {
+      font-size: 14px;
+      font-weight: bold;
+
+      span {
+        padding-right: 2px;
+      }
+
+      .legend-expand {
+        width: auto;
+        float: right;
+        padding: 0;
+        margin: 0;
+        color: black;
+        border: none;
+        border-radius: 0;
+        box-shadow: none;
+
+        i {
+          padding: 2px;
+          background-color: color('base-lighter');
+        }
+      }
+    }
+
+    .legend-list-container {
+      #network-legend-list {
+        span {
+          margin-left: 2px;
+        }
+
+        img {
+          padding-top: 2px;
+          width: 15px;
+        }
+      }
+
+      #fim-legend-list {
+        span {
+          margin-left: 2px;
+        }
+
+        img {
+          padding-top: 2px;
+          width: 15px;
+          height: 13px;
+        }
+      }
+    }
+
+  }
+
+  .leaflet-control-layers-selector {
+    left: 0;
+    position: relative;
+    display: inline;
+    height: auto;
+    width: auto;
+    outline-style: none;
+  }
+
+  label {
+    margin: auto;
+  }
+}
+
+sort {
```

Comment:
Are these actually used? I'm wondering if they got brought in with one of the other table libraries. If the styles are not used please remove. I doubt they are because this is a style for a "sort" tag

Reply:
I went ahead and deleted the sort tag from the CSS